verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017-2020 The Verible Authors.
```

Comment:
Please add a unit test.

Reply:
Tip: the `VectorTree` class lets you construct expected structures that can be compared against.
All you need is to be able to construct ValueType.
See examples in https://github.com/google/verible/blob/master/common/util/vector_tree_test.cc.

```c++
typedef VectorTree<Value> T;

const auto expected_tree = T(value1, T(...), T(...));
// produce an actual tree
// use DeepEqual(actual_tree, expected_tree) ...
```